cmd/mesh/manifest-diff.go

```diff
@@ -58,10 +58,15 @@ func addManifestDiffFlags(cmd *cobra.Command, diffArgs *manifestDiffArgs) {
 
 func manifestDiffCmd(rootArgs *rootArgs, diffArgs *manifestDiffArgs) *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "diff",
+		Use:   "diff <file|dir> <file|dir>",
 		Short: "Compare manifests and generate diff.",
 		Long:  "The diff subcommand compares manifests from two files or directories.",
-		Args:  cobra.ExactArgs(2),
+		Args: func(cmd *cobra.Command, args []string) error {
```

Comment:
shouldn't cobra.ExactArgs(2) does the number of arguments check already?

Reply:
@richardwxn There is a general request from @sdake to explain the problem.  See https://github.com/istio/istio/issues/16575

ExactArgs(2) outputs `Error: requires at least 1 arg(s), only received 0`.  The new code outputs `Error: diff requires two files or directories` then a usage message that includes `diff <file|dir> <file|dir>`.